### PR TITLE
website: index page for installation guide

### PR DESCRIPTION
**Pull request by deekay2310**
Signed-off-by: Dev Kumar <devskumar99@gmail.com>

### Screenshot/screencast of this PR
Screen Shot 2022-11-01 at 11 14 57 AM

### CODE REVIEW

File: website/docs/Installation/index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+layout: default
+title: Installation Guide for Podman Desktop
+---
+
+Welcome to the Installation Guide for Podman Desktop!
+
+Podman Desktop is available across all platforms and this documentations intends to help you install the latest version of Podman Desktop on your Operating System. The links to installation guide for different platforms are listed below.
```
Comment (benoitf) [inline website/docs/Installation/index.md]: I think 'all platforms' is misleading as it's only Mac arm64/x64, win x64 and Linux x64
But all platforms include several other operating systems / arches

Comment (deekay2310) [reply website/docs/Installation/index.md]: should I replace it with "across Windows, MacOS and Linux"?

Comment (benoitf) [reply website/docs/Installation/index.md]: maybe

Comment (deekay2310) [reply website/docs/Installation/index.md]: done

**Comment by deekay2310** [comment]
Screen Shot 2022-11-01 at 12 59 29 PM

**Comment by deekay2310** [comment]
Screen Shot 2022-11-01 at 1 01 26 PM

**Pull request merged**